_i18n/en.yml

```diff
@@ -559,6 +559,20 @@ mining:
   software_para: Note that some miners may have developer fees.
   support: Support
   supportp: If you have questions or just want to confront with fellow miners, come chat on Monero Pools. On <a href="#monero-pools:monero.social">Matrix</a> and <a href="irc://irc.libera.chat/#monero-pools">Libera</a>.
+  p2poolh: "P2Pool: The best of both solo and pool mining"
+  p2poolnew: >
+    P2Pool is a new clever way of mining Monero, which allows to receive frequent payouts like if you are mining in a pool, but while mining solo. It's a Peer-To-Peer mining pool that gives you full control over your Monero node and what it mines. More details in <a href="{{ site.baseurl }}/2021/10/05/p2pool-released.html">the announcement post.
+  p2poolp: >
+    P2Pool is a sidechain to Monero, and P2Pool blocks are potentially Monero blocks. Each miner submits block templates that include a payout for all of the miners that currently have shares in the PPLNS window. High quality block templates are added to the P2Pool blockchain as blocks which count as "shares" for the miner who found them.
+  p2poolfeatures: Main features
+  p2pdecentralized: <b>Decentralized:</b> no central server that can be shutdown/blocked.
+  p2ppermissionless: <b>Permissionless:</b> there is no one to decide who can mine on the pool and who can't.
```

Comment:
Permissionless: anyone can join the sidechain, and it's effectively impossible to censor individual miners.